tests/boot/snapper_rollback.pm

```diff
@@ -22,7 +22,7 @@ use version_utils;
 
 sub run {
     my ($self) = @_;
-    if (is_leap_migration && (check_var('DESKTOP', 'gnome') || check_var('DESKTOP', 'kde'))) {
+    if ((is_leap_migration || is_opensuse) && (check_var('DESKTOP', 'gnome') || check_var('DESKTOP', 'kde'))) {
```

Comment:
`is_leap_migration` should imply "openSUSE" by the name. `is_opensuse` should not be needed

Reply:
> `is_leap_migration` should imply "openSUSE" by the name. `is_opensuse` should not be needed
is_leap_migration is for leap to sle migration. here we just make sure both leap to sle migration and OpenSuse tests with KDE and gnome desktop will boot directly into desktop.